tests/suites/test_suite_pk.function

```diff
@@ -434,7 +434,7 @@ void pk_rsa_encrypt_test_vec( data_t * message, int mod, int radix_N,
                               char * input_N, int radix_E, char * input_E,
                               data_t * result, int ret )
 {
-    unsigned char output[1000];
+    unsigned char output[300];
```

Comment:
How are we guaranteeing that 300 is enough?

Reply:
I ran the tests on PC and printed the values.